packages/features/form-builder/Components.tsx

```diff
@@ -489,10 +489,11 @@ export const Components: Record<FieldType, Component> = {
   },
   boolean: {
     propsType: propsTypes.boolean,
-    factory: ({ readOnly, label, value, setValue }) => {
+    factory: ({ readOnly, name, label, value, setValue }) => {
       return (
         <div className="flex">
           <CheckboxField
+            name={name}
```

Comment:
We weren't passing name. It is required by tests to verify fields and is otherwise useful as well.